packages/vue/src/components/password-control.vue

```diff
@@ -12,15 +13,17 @@ const random = Math.floor(Math.random() * 999999);
 interface PropsInterface {
   name: string;
   label: string;
-  autocomplete: string;
-  hasError: boolean;
+  autocomplete?: string;
+  hasError?: boolean;
```

Comment:
Made these optional, but with defaults, to match React and Angular.

This is not breaking, because (1) this is not a public component and (2) it's going `required` -> `optional` so previous usages always provided this prop anyway.